src/WCSimTrajectory.cc

```diff
@@ -68,7 +68,7 @@ WCSimTrajectory::WCSimTrajectory(const G4Track* aTrack)
       creatorProcess = tempproc->GetProcessName();
     }
   else 
-    creatorProcess = "";
+    creatorProcess = "NONE";
```

Comment:
maybe go back to use `""` instead of `"NONE"`

Reply:
👍